src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -632,6 +632,11 @@ want to solicit feedback from the community.
 
     Added in `1.8.0`.
 
+    When set to `disk` along with setting
+    `OTEL_DOTNET_EXPERIMENTAL_OTLP_DISK_RETRY_DIRECTORY_PATH` to the path on
+    disk, it enables retries by storing telemetry on disk during transient
+    errors. TODO: Add package version.
```

Comment:
nit: Let's tweak this a bit. I think bullets would help and also instead of "TODO" let's say it is unreleased:

* All signals

  * `OTEL_DOTNET_EXPERIMENTAL_OTLP_RETRY`

    * When set to `in_memory`, it enables in-memory retry for transient errors
       encountered while sending telemetry.

       Added in `1.8.0`.

    * When set to `disk` along with setting
       `OTEL_DOTNET_EXPERIMENTAL_OTLP_DISK_RETRY_DIRECTORY_PATH` to the path on
       disk, it enables retries by storing telemetry on disk during transient
       errors.

       Added in **TBD** (Unreleased)

Reply:
updated